langchain/scripts/create-entrypoints.js

```diff
@@ -76,6 +76,7 @@ const entrypoints = {
   "llms/load": "llms/load",
   "llms/base": "llms/base",
   "llms/openai": "llms/openai",
+  "llms/azure_openai": "llms/azure_openai",
```

Comment:
Let's not export this from `langchain` - we're deprecating integration exports from the main package in favor of the integrations packages. Can you revert all the LangChain changes?

Reply:
Per the comments, **ALL** the changes from the langchain package have been reverted back in the latest commit.